components/l10n/common/ofac_sanctioned_iso_3166_1_country_code_constants.h

```diff
@@ -0,0 +1,53 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_L10N_COMMON_OFAC_SANCTIONED_ISO_3166_1_COUNTRY_CODE_CONSTANTS_H_
+#define BRAVE_COMPONENTS_L10N_COMMON_OFAC_SANCTIONED_ISO_3166_1_COUNTRY_CODE_CONSTANTS_H_
+
+#include <string>
+
+#include "base/containers/flat_set.h"
+
+namespace brave_l10n {
+
+// See https://orpa.princeton.edu/export-controls/sanctioned-countries.
+
+const base::flat_set<std::string> kOFACSactionedISO31661CountryCodes = {
```

Comment:
This actually should be a `fixed_flat_set`, which will be `constexpr`.

Reply:
Fixed, thanks